src/datasources/config-api/config-api.service.ts

```diff
@@ -59,9 +59,11 @@ export class ConfigApi implements IConfigApi {
 
   async clearChains(): Promise<void> {
     const pattern = CacheRouter.getChainsCachePattern();
+    const key = CacheRouter.getChainsCacheKey();
     await Promise.all([
-      this.cacheService.deleteByKey(CacheRouter.getChainsCacheKey()),
+      this.cacheService.deleteByKey(key),
       this.cacheService.deleteByKeyPattern(pattern),
+      // TODO: call _setInvalidationTimeForKey for each item matching the pattern
```

Comment:
This function needs to be reviewed as the `flush` endpoint shouldn't be used anymore anyway.

Reply:
Yes, I agree this function shouldn't be necessary once the `flush` endpoint is deprecated. We could review it in a separate issue if you agree 🙂